tools/pyinstaller_hooks/hook-scalecodec.py

```diff
@@ -1,5 +0,0 @@
-from PyInstaller.utils.hooks import collect_data_files
```

Comment:
by deleting this you **have to** check it works packaged for kusama. Otherwise we may end up having problems again at release time